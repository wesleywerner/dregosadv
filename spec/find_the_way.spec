superbrief
gonear clearing portal
go east
go west
go northeast
read compass
climb the tree
look northeast
put compass in bag
climb down
go north
