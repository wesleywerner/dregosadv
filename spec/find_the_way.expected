  THIS GAME IS IN DEBUG MODE
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 400410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
  The a canoe rocks gently on the water as you travel up the river. Your
  guide sits at the other end of the craft rowing with an oar. You wonder how
  much longer this will take, you have been in this a canoe for a while now.
  You are anxious to start searching for the treasure.
  
  In the canoe you can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  Dr Ego and the Egg of Man-Toomba is now in its "superbrief" mode, which
  always gives short descriptions of locations (even if you haven't been
  there before).
  
> >  Clearing
  
  A hut is due south.
  A jungle path is due east.
> >  ; == 82
  
> >  Lost in the Jungle
  
> >  Your compass guides you back to safety.
  
  Clearing
  
  A hut is due south.
  A jungle path is due east.
  [The score has just gone up by one point.]
  
> >  You climb up the strong branches of the fig tree.
  
  Top of Tree
  
  The monkey is not impressed, it climbs down the tree.
  
> >  There! Stone stairs up the hill, in the distance to the northeast.
  
  [The score has just gone up by one point.]
  
> >  As you descend you drop your compass to the ground below. CLINK!
  
   --- more ---
)   Clearing
  
  A hut is due south.
  A jungle path is due east.
  A valley is due northeast.
  There is a monkey here. It has your compass.
  
  The monkey makes grunting noises.
  
> >  Valley
  
  A foot path is due east.
  Some stone stairs leads upward.
  A clearing in the jungle is due south.
> >